cinema_service/manage.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+from typing import NoReturn
+
+
+def main() -> NoReturn:
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'cinema_service.settings')
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError as exc:
+        raise ImportError(
+        ) from exc
```

Comment:
Improvement Suggestion: Add a message to the `ImportError` to provide more context, such as 'Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable?'